pkg/kv/kvserver/closedts/setting.go

```diff
@@ -16,33 +16,41 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/settings"
 )
 
+// NB: These settings are TenantReadOnly because they need to be read by e.g.
+// rangefeed clients and follower_read_timestamp(). However, they really need
+// to see the host's setting, not the tenant's setting. See:
+// https://github.com/cockroachdb/cockroach/issues/108677
+
 // TargetDuration is the follower reads closed timestamp update target duration.
 var TargetDuration = settings.RegisterDurationSetting(
-	settings.TenantWritable,
+	settings.TenantReadOnly,
```

Comment:
Do we need to call this out as a breaking change? If users are erroneously setting these in their tenants in automation, they will now get an error.

Reply:
It only affects serverless users. Not sure how we communicate breaking changes there, but release notes don't seem like the best channel. Will find out.

Reply:
Added a release note.